pytorch_lightning/plugins/native_amp.py

```diff
@@ -59,3 +59,7 @@ def training_step(self, fx, args):
     def clip_gradients(self, grad_clip_val: Union[int, float], optimizer: Optimizer, norm_type: float):
         model = self.trainer.get_model()
         torch.nn.utils.clip_grad_norm_(model.parameters(), max_norm=grad_clip_val, norm_type=norm_type)
+
+    @property
```

Comment:
doesn’t native amp also need this defined? otherwise it will crash when it tries to get the property

Reply:
This is native amp, no custom amp yet...